### PR TITLE
Update references from back link to inset text in inset text options list

### PR DESCRIPTION
**Pull request by matteason**
Just fixes a couple of copy/paste errors that have resulted in the inset text guidance on the design system containing references to the back link component by mistake

### CODE REVIEW

**Review by domoscargin — APPROVED** [review]
Nice spot!

Needs a second review.

**Review by colinrotherham — APPROVED** [review]

**Pull request merged**
